src/components/sbb-timetable-occupancy-icon/sbb-timetable-occupancy-icon.ts

```diff
@@ -0,0 +1,89 @@
+import { CSSResult, html, LitElement, PropertyValues, TemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import style from './sbb-timetable-occupancy-icon.scss?lit&inline';
+import { OccupancyEnum } from '../core/timetable';
+import { documentLanguage, HandlerRepository, languageChangeHandlerAspect } from '../core/eventing';
+import { i18nOccupancy } from '../core/i18n';
+import '../sbb-icon';
+
+@customElement('sbb-timetable-occupancy-icon')
+export class SbbTimetableOccupancyIcon extends LitElement {
+  public static override styles: CSSResult = style;
+
+  /** Wagon occupancy. */
+  @property() public occupancy!: OccupancyEnum;
```

Comment:
We prefer to not use uppercase and enums ;). Maybe let's discuss this again, especially which term is correct ? none vs. unknown.

Reply:
We already have a SbbOccupancy in index.d.ts (lowercase, type so not an enum), we can change to it (but we need to warn consumers and maybe rename some icons for a clean work)